ic-asset/src/sync.rs

```diff
@@ -49,12 +49,21 @@ pub async fn sync(
     Ok(())
 }
 
-fn filename_starts_with_dot(entry: &walkdir::DirEntry) -> bool {
-    entry
+fn doesnt_start_with_a_dot_or_explicitly_included(
+    entry: &walkdir::DirEntry,
+    config: &AssetSourceDirectoryConfiguration,
+) -> bool {
+    let starts_with_a_dot = entry
```

Comment:
Suggest reversing the meaning of this method (return true to include the entry)
```suggestion
fn include_entry(
    entry: &walkdir::DirEntry,
    config: &AssetSourceDirectoryConfiguration,
) -> bool {
    let starts_with_a_dot = entry
```